src/DataSource/DB.hs

```diff
@@ -22,18 +22,20 @@ module DataSource.DB
 
 import           Universum
 
+import           Control.Exception.Safe (Handler(..), catches)
 import           Control.Monad.Trans.Control (MonadBaseControl)
 
 import           Data.Pool (Pool, createPool, withResource)
 import           Data.Text (split)
 
 import           Database.SQLite.Simple (FromRow (..), NamedParam (..), SQLData (..), close,
-                                         executeNamed, execute_, field, open, queryNamed, query_)
+                                         executeNamed, execute_, field, open, queryNamed, query_, FormatError(..))
```

Comment:
This is just from my eyes :eye:, but it's not formatted?

Reply:
Good catch! I'll fix it